### PR TITLE
Add aspect ratio support

### PR DESCRIPTION
**Pull request by sigvef**
<h4>Add aspect ratio support</h4>

Now demos can have custom aspect ratios.

### CODE REVIEW

File: renin/src/renin.ts [hunk 1]
```diff
@@ -60,6 +60,7 @@ export interface Options {
   productionMode: boolean;
   rendererOptions?: WebGLRendererParameters;
   toneMapping: WebGLRenderer['toneMapping'];
+  aspectRatio: number;
```
Comment (stianjensen) [inline renin/src/renin.ts:63]: Does this need a default for demos who haven't set it?

**Review by stianjensen — CHANGES_REQUESTED** [review]

File: renin/src/renin.ts (hunk 1 above)
Comment (trondkla) [reply renin/src/renin.ts:64]: Should be default 1?

**Review by trondkla — APPROVED** [review]
Looks good, but should have a default value as stian said

File: renin/src/renin.ts (hunk 1 above)
Comment (trondkla) [reply renin/src/renin.ts:64]: Sat the aspectRatio to 16.0/9 in the example-project instead after a discussion on messenger

**Review by trondkla — APPROVED** [review]

**Pull request merged**